schema/compose-spec.json

```diff
@@ -330,7 +330,7 @@
         "privileged": {"type": "boolean"},
         "profiles": {"$ref": "#/definitions/list_of_strings"},
         "pull_policy": {"type": "string", "enum": [
-          "always", "never", "if_not_present"
+          "always", "never", "missing", "build"
```

Comment:
I don't know any implementation to use it, but as `if_not_present` was in the spec it should not be removed. Maybe we should just declare this as an alias

You also need to define those new values in https://github.com/compose-spec/compose-spec/blob/master/spec.md#pull_policy

Reply:
I'll leave the change on if_not_present for another PR then, my point was to add "build". 
Updated the md specs, thx